File: Pattern2.java
public class Pattern2 
{
    public static void Print(int ino)
    {
       for(int i = ino ; i>0 ; i--)
       {
        for(int j = 1 ; j <= i ; j++)
        {
            System.out.print(j+" ");
        }
        System.out.println();
       }
    }

    public static void main(String[] args) 
    {
        int ino = 7;

        Print(ino);
        
    }

    
}

File: Pattern2.c
// Input : 5
// Output : -5 -4 -3 -2 -1


#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
   
    for(iCnt = -iNo; iCnt < 0; iCnt++)
    {
        printf("%d\t ",iCnt);   
    }    
}
int main()
{
    int iFrequency = 0;

    printf("ENter the frequency of Symbol : \n");
    scanf("%d",&iFrequency);

    Display(iFrequency);

    return 0;
}